TLCALIPER-11 - Add installation/usage instructions to README.md

lsloan opened the pull request:
Added instructions for using Composer to install Caliper.  Corrected other usage instructions as well.

Hint: For easier reviewing, click the `View` button beside the heading for `README.md` in the list on the `Files changed` tab.

**_NOTE**_ - By following the updated instructions in `README.md`, we should be able to use Composer to add caliper-php to the Problem Roulette project.  It will no longer be necessary to include caliper-php directly in the project.

Reviewers: @bkirschn @arwhyte @pushyamig

lsloan committed:
TLCALIPER-11 - Added instructions for using Composer to install Caliper.

lsloan committed:
TLCALIPER-11 - Slight working changes.

pushyamig commented on README.md:
````diff
@@ -1,55 +1,137 @@
+### Pre-requisites for development
 
 * PHP 5.4 required (PHP 5.6 recommended)
 * Ensure you have php5 and php5-json installed:  ```sudo apt-get install php5 php5-json```
````
I think this line is redundant as the before line says that PHP5.4 is required and php5-json is added to php from 5.2 version onwards. It will confuse people

lsloan replied:
Good point.  Corrected.

lsloan committed:
Removed requirement about "php5" and "php5-json"

As pointed out during code review, the JSON extension for PHP has been compiled into the interpreter since PHP 5.2.0.  Removing this line to eliminate confusion and frustration.
